classes/search/ArticleSearchDAO.inc.php

```diff
@@ -80,8 +80,12 @@ public function getPhraseResults($journal, $phrase, $publishedFrom = null, $publ
 				JOIN publication_settings ps ON (ps.publication_id = p.publication_id AND ps.setting_name=\'issueId\' AND ps.locale=\'\')
 				JOIN issues i ON (CAST(i.issue_id AS CHAR(20)) = ps.setting_value AND i.journal_id = s.context_id)
 				JOIN submission_search_objects o ON (s.submission_id = o.submission_id)
+				JOIN journals con ON con.journal_id = s.context_id
+				LEFT OUTER JOIN journal_settings con_s_publishingMode on con.journal_id = con_s_publishingMode.journal_id AND con_s_publishingMode.setting_name = \'publishingMode\'
 				NATURAL JOIN ' . $sqlFrom . '
 			WHERE
+				(con_s_publishingMode.setting_value <> \'' . PUBLISHING_MODE_NONE . '\' OR
```

Comment:
Do we need `'` around the constant PUBLISHING_MODE_NONE ?

Reply:
I am using `'` because the `js.setting_value` is a `varchar` column and the value to be compared should be varchar one, thus the `'` enclosure of the `PUBLISHING_MODE_NONE`.